tensilelite/Tensile/TensileCreateLibrary/Run.py

```diff
@@ -152,6 +157,18 @@ def removeInvalidSolutionsAndKernels(results, kernels, solutions, errorTolerant,
     for rel in removeResults:
         results.remove(rel)
 
+def passPostKernelInfoToSolution(results, kernels, solutions, splitGSU: bool):
+    for solution in solutions:
+        solutionKernels = solution.getKernels()
+        for kernel in solutionKernels:
+            kName = getKeyNoInternalArgs(kernel, splitGSU)
+            for kernIdx, r in enumerate(results):
+                if kName == getKeyNoInternalArgs(kernels[kernIdx], splitGSU):
+                    solution._state["CUOccupancy"] = r.cuoccupancy
+                    solution._state["PrefetchGlobalRead"] = r.pgr
+                    solution._state["MathClocksUnrolledLoop"] = r.mathclk
+                    break
+            break
```

Comment:
Need to speedup here by using dictionary as lookup table instead of comparing 2 of arrays